Snowflake.Data/Core/ArrowResultChunk.cs

```diff
@@ -111,6 +144,14 @@ public override UTF8Buffer ExtractCell(int columnIndex)
             throw new NotSupportedException();
         }
 
+        public string GetMetadata()
```

Comment:
I would remove this function, it was for testing purpose.

Reply:
Done